ethereum/core/src/test/java/tech/pegasys/teku/core/signatures/LocalSignerTest.java

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright 2020 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.teku.core.signatures;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.apache.tuweni.bytes.Bytes;
+import org.junit.jupiter.api.Test;
+import tech.pegasys.teku.bls.BLSKeyPair;
+import tech.pegasys.teku.bls.BLSSignature;
+import tech.pegasys.teku.datastructures.blocks.BeaconBlock;
+import tech.pegasys.teku.datastructures.operations.AggregateAndProof;
+import tech.pegasys.teku.datastructures.operations.AttestationData;
+import tech.pegasys.teku.datastructures.operations.VoluntaryExit;
+import tech.pegasys.teku.datastructures.state.ForkInfo;
+import tech.pegasys.teku.datastructures.util.DataStructureUtil;
+import tech.pegasys.teku.infrastructure.async.SafeFuture;
+import tech.pegasys.teku.infrastructure.async.StubAsyncRunner;
+import tech.pegasys.teku.infrastructure.unsigned.UInt64;
+
+class LocalSignerTest {
+  private final DataStructureUtil dataStructureUtil = new DataStructureUtil();
+  private final ForkInfo fork = dataStructureUtil.randomForkInfo();
+
+  private static final BLSKeyPair KEYPAIR = BLSKeyPair.random(1234);
+  private final StubAsyncRunner asyncRunner = new StubAsyncRunner();
+
+  private final LocalSigner signer = new LocalSigner(KEYPAIR, asyncRunner);
+
+  @Test
+  public void shouldSignBlock1() {
```

Comment:
This name seems odd, shouldSignBlock?

Reply:
copied over from now deleted "SignerTest". Let me fix it.